src/index.tsx

```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
```

Comment:
please re-arrange imports.

Imports should be grouped in the following order:
* standard library imports
* related third party imports
* local application/library specific imports